dotcom-rendering/src/components/YoutubeAtom/YoutubeAtom.stories.tsx

```diff
@@ -178,56 +174,25 @@ export const NoOverlay = {
 	},
 } satisfies Story;
 
-export const WithOverrideImage = {
```

Comment:
The decision on which image to use, `posterImage` or `overrideImage`, is moved up to YoutubeBlockComponent.importable.tsx, so we no longer need many of these these stories.